gdx/src/com/badlogic/gdx/Screen.java

```diff
@@ -16,14 +16,16 @@
 
 package com.badlogic.gdx;
 
+import com.badlogic.gdx.utils.Disposable;
+
 /** <p>
  * Represents one of many application screens, such as a main menu, a settings menu, the game screen and so on.
  * </p>
  * <p>
  * Note that {@link #dispose()} is not called automatically.
  * </p>
  * @see Game */
-public interface Screen {
+public interface Screen extends Disposable {

```

Comment:
I think not all Screens must be Disposable. Also this will break every existing code which uses Screen and doesn't implement Disposable.

Reply:
It has been restored back it was. 👍